### PR TITLE
Searchwork take II. Fixes #795. Also fixes #834

### LINKED ISSUE #795: [Discussion] clicking several labels on issues list should accumulate, not replace them
When you load a list of issues you may want to easily see all "Needs contact" for "IE", or all "Needs diagnosis" for "firefox". You can click labels for various workflow steps (like "Needs contact") at the top, and you can click labels like "firefox" next to the bug summary. However, if you click first "Needs contact", then "firefox", the latter action will cancel out the first one.

The way the UI is organised indicates strongly that it should be possible to combine at least one of the top-level workflow statuses with some other tag or search. It would be useful to make that work.

Personally I'd suggest that clicking other labels _always_ should accumulate them, and I'd also like to see a small UI where selected labels appear and can be "deleted" again with a little x - this is a common UI idiom when searching by labels or keywords now.

So @webcompat/owners and @webcompat/collaborators - please discuss :)

### LINKED ISSUE #834: Issues JS generates self-contradicting backend URLs
Initial URL is fine:
https://webcompat.com/api/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc
The one you get by clicking "new" has a "stage=all" in it:
https://webcompat.com/api/issues/search/new?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc&q=netflix
and if you keep clicking those labels, you'll see that "stage" keeps lagging one step behind and contradicting the path:

https://webcompat.com/api/issues/category/needsdiagnosis?page=1&per_page=50&state=open&stage=new&sort=created&direction=desc

https://webcompat.com/api/issues/category/needscontact?page=1&per_page=50&state=open&stage=needsdiagnosis&sort=created&direction=desc

Hmm.. :-/

### PR DESCRIPTION
**Pull request by hallvors**
Adding another "searchwork" PR for review and discussion :)

### CODE REVIEW

File: grunt-tasks/concat.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = function(grunt) {
             '<%= jsPath %>/vendor/prism.js',
             '<%= jsPath %>/vendor/markdown-it.js',
             '<%= jsPath %>/vendor/markdown-it-emoji-1.0.0.js',
-            '<%= jsPath %>/vendor/markdown-it-sanitizer-0.3.0.js',
+            '<%= jsPath %>/vendor/markdown-it-sanitizer-0.4.1.js',
```
Comment (hallvors) [inline grunt-tasks/concat.js]: Oops, sorry. That's part of a different fix.. :p

**Comment by miketaylr** [comment]
Tests didn't run because of failing eslint check. I'd like us to discuss these changes in person as a team before even looking at this. Let's do today.

**Comment by hallvors** [comment]
Ops, will do eslint.. :/

**Comment by hallvors** [comment]
Not quite ready yet, closing.

**Pull request closed without merging**